runtime/reference/ts_config_migration.md

```diff
@@ -0,0 +1,501 @@
+| `noUnusedLocals`                 | `false`                 |                                                                                                                                           |
+| `noUnusedParameters`             | `false`                 |                                                                                                                                           |
+| `noUncheckedIndexedAccess`       | `false`                 |                                                                                                                                           |
+| `reactNamespace`                 | `React`                 |                                                                                                                                           |
+| `strict`                         | `true`                  |                                                                                                                                           |
+| `strictBindCallApply`            | `true`                  |                                                                                                                                           |
+| `strictFunctionTypes`            | `true`                  |                                                                                                                                           |
+| `strictPropertyInitialization`   | `true`                  |                                                                                                                                           |
+| `strictNullChecks`               | `true`                  |                                                                                                                                           |
+| `suppressExcessPropertyErrors`   | `false`                 |                                                                                                                                           |
+| `suppressImplicitAnyIndexErrors` | `false`                 |                                                                                                                                           |
+| `useUnknownInCatchVariables`     | `false`                 |                                                                                                                                           |
+
+For a full list of compiler options and how they affect TypeScript, please refer
+to the
+[TypeScript Handbook](https://www.typescriptlang.org/docs/handbook/compiler-options.html).
+
+## Using the "lib" property
+
+Deno includes unique built-in libraries not found in other platforms (like
+`tsc`). This can sometimes pose challenges when writing code meant to run in
+both Deno and a browser. In such cases, you can tweak Deno’s behavior using the
+"lib" property within your `compilerOptions`.
```

Comment:
Nit: There shouldn't be "challenges" with Deno when it comes to the wording. Maybe something like:

```suggestion
If you're working on a project that ships code to multiple runtimes, like browsers for example, you can tweak the default types via the "lib" property within the `compilerOptions`.
```